_posts/2017/2017-02-23-why-i-love-working-as-a-scrum-master.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Why I love working as a Scrum Master"
+date: "2017-02-23 07:00:00"
+categories: working-at-epages
```

Comment:
cool! Is she fine with the content?
With regards to the picture of her: please ensure it fits on both pages of the blog - the overview page and the detail page - so that face, head, whatever ;-) is not cut.